sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CastSuiteBase.scala

```diff
@@ -73,7 +73,9 @@ abstract class CastSuiteBase extends SparkFunSuite with ExpressionEvalHelper {
     }
   }
 
-  protected def isAlwaysNullable: Boolean = false
+  // Whether the test suite is for TryCast. If yes, there is no exceptions and the result is
+  // always nullable.
+  protected def isTryCast: Boolean = false
```

Comment:
In this case, this looks like a major refactoring instead of simple **moving**.

Reply:
yea we should mention it in the PR description

Reply:
+1, PR description updated.